### PR TITLE
fix: the client ui

### PR DESCRIPTION
**Pull request by floralatin**

### CODE REVIEW

**Comment by delgermurun** [comment]
Could you please add some context? Why it is needed?

**Comment by floralatin** [comment]
@delgermurun  Could you quickly review this PR, thank you very much.

**Comment by floralatin** [comment]
> Could you please add some context? Why it is needed?

In version v0.23.1  when the client inits the property username, it will request the hubble backend. and it will throw the error, as the picture shows:
image

File: hubble/client/base.py [hunk 1]
```diff
@@ -60,6 +60,7 @@ def handle_request(
         files: Optional[MutableMapping[Text, IO[Any]]] = None,
         headers: Optional[dict] = None,
         json: Optional[dict] = None,
+        print_logger: Optional[bool] = True,
```
Comment (delgermurun) [inline hubble/client/base.py:63]:
```suggestion
        log_error: Optional[bool] = True,
```

**Review by delgermurun — CHANGES_REQUESTED** [review]

**Review by delgermurun — APPROVED** [review]
LGTM

**Pull request merged**